docs/contributing/release_process.md

```diff
@@ -45,101 +93,126 @@ Use the following template for a clean changelog after the release:
 
 <!-- Include any especially major or disruptive changes here -->
 
-### Style
+### Stable style
 
 <!-- Changes that affect Black's stable style -->
 
 ### Preview style
 
 <!-- Changes that affect Black's preview style -->
 
-### _Blackd_
-
-<!-- Changes to blackd -->
-
 ### Configuration
 
 <!-- Changes to how Black can be configured -->
 
-### Documentation
+### Packaging
 
-<!-- Major changes to documentation and policies. Small docs changes
-     don't need a changelog entry. -->
+<!-- Changes to how Black is packaged, such as dependency requirements -->
 
-### Integrations
+### Parser
 
-<!-- For example, Docker, GitHub Actions, pre-commit, editors -->
+<!-- Changes to the parser or to version autodetection -->
+
+### Performance
+
+<!-- Changes that improve Black's performance. -->
 
 ### Output
 
 <!-- Changes to Black's terminal output and error messages -->
 
-### Packaging
-
-<!-- Changes to how Black is packaged, such as dependency requirements -->
+### _Blackd_
 
-### Parser
+<!-- Changes to blackd -->
 
-<!-- Changes to the parser or to version autodetection -->
+### Integrations
 
-### Performance
+<!-- For example, Docker, GitHub Actions, pre-commit, editors -->
 
-<!-- Changes that improve Black's performance. -->
+### Documentation
 
+<!-- Major changes to documentation and policies. Small docs changes
+     don't need a changelog entry. -->
 ```
 
 ## Release workflows
 
-All _Blacks_'s automation workflows use GitHub Actions. All workflows are therefore
-configured using `.yml` files in the `.github/workflows` directory of the _Black_
+All of _Black_'s release automation use [GitHub Actions]. All workflows are therefore
+configured using YAML files in the `.github/workflows` directory of the _Black_
 repository.
 
+They are triggered by the publication of a [GitHub Release].
+
 Below are descriptions of our release workflows.
 
-### Docker
+### Publish to PyPI
 
-This workflow uses the QEMU powered `buildx` feature of docker to upload a `arm64` and
-`amd64`/`x86_64` build of the official _Black_ docker image™.
+This is our main workhorse workflow. It builds a [sdist] and [wheels] to upload to PyPI
+where the vast majority of users will download Black from. It's divided into three job
+groups:
 
-- Currently this workflow uses an API Token associated with @cooperlees account
+#### sdist + pure wheel
 
-### pypi_upload
+This single job builds the sdist and pure Python wheel (ie. a wheel that only contains
+Python code) using [build]. These artifacts are general-purpose and can be used on
+basically any platform supported by Python.
 
-This workflow builds a Python
-[sdist](https://docs.python.org/3/distutils/sourcedist.html) and
-[wheel](https://pythonwheels.com) using the latest
-[setuptools](https://pypi.org/project/setuptools/) and
-[wheel](https://pypi.org/project/wheel/) modules.
+#### mypyc wheels (…)
```

Comment:
What is the ellipsis for?

Reply:
Supposed to be a placeholder for the names the matrix jobs will have in practice. 
![image](https://user-images.githubusercontent.com/63936253/187310718-df6d606f-9b90-4363-925b-ed4afff334d0.png)

I'm happy to remove them if you find them confusing.